Dockerfile

```diff
@@ -1,7 +1,23 @@
-FROM clojure:tools-deps-alpine
-COPY --from=cljkondo/clj-kondo:2022.05.31-alpine /bin/clj-kondo /usr/local/bin/clj-kondo
+FROM ghcr.io/graalvm/native-image:22.2.0 AS build-clj-kondo
 
-RUN apk add git
+RUN microdnf install -y gzip tar && \
+    curl -LO  https://github.com/clj-kondo/clj-kondo/archive/refs/tags/v2022.08.03.tar.gz && \
+    gunzip v2022.08.03.tar.gz && \
+    tar -xvf v2022.08.03.tar
+
+WORKDIR /app/clj-kondo-2022.08.03/
+ENV GRAALVM_HOME /usr
+
+RUN curl -o /usr/local/bin/lein https://raw.githubusercontent.com/technomancy/leiningen/stable/bin/lein && \
+    chmod +x /usr/local/bin/lein
```

Comment:
Is there any reason to take a different way of installing Leiningen from the one for the `clojure:tools-deps-jammy` container below? I think either way (`RUN curl` or `ADD`) works, but it would be good to apply a consistent way to both images if possible.